payment/serializers.py

```diff
@@ -118,18 +118,21 @@ def create(self, validated_data):
             )
 
         payment_data = {
-            "parent_name": payment.parent.user.first_name,
+            "parent_name": invoice.parent.user.first_name,
             "business_name": settings.BUSINESS_NAME,
-            "receipt_text": f"You've paid {payment.total} for {len(registrations)} classes",
-            "total_tuition": float(payment.total),
-            "payment_id": payment.id,
+            "receipt_text": f"You've paid {invoice.total} for {len(registrations)} classes",
+            "total_tuition": float(invoice.total),
+            "payment_id": invoice.id,
             "enrollments": {
                 "course": [{
                     "title": registration.enrollment.course.title,
                     "start_date": registration.attendance_start_date.strftime("%Y-%m-%d"),
                     "end_date": registration.enrollment.course.end_date.strftime("%Y-%m-%d"),
-                    "start_time": registration.enrollment.course.start_time.strftime("%H:%M"),
-                    "end_time": registration.enrollment.course.end_time.strftime("%H:%M"),
+                    # TODO migrate sendgrid template to availability list 
```

Comment:
Check out the serializer code in my PR for a fix. Please copy it over and see if it works with your template.

Reply:
Resolved! I also tested to see if the email worked, and we are good to go!